fenix/app/src/nightly/res/xml/shortcuts.xml

```diff
@@ -4,6 +4,17 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 
 <shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:shortcutId="password_manager"
+        android:enabled="true"
+        android:icon="@drawable/ic_static_shortcut_password_manager"
```

Comment:
Same issue here. This should be `android:icon="@drawable/ic_static_password_shortcut"`